src/api/typings/coinmarketcap.d.ts

```diff
@@ -0,0 +1,9 @@
+interface Prices {
+  [key: string]: number
+}
+
+/** Returns the price of coin in the symbol given */
+export function getPrice(coin?: string, currency?: string): Promise<number>
```

Comment:
Following your convention, this should get a `DEPRECATED` tag

Reply:
added